apps/android_rpc/app/src/main/jni/tvm_runtime.h

```diff
@@ -34,3 +37,10 @@
 #include "../src/runtime/vulkan/vulkan_device_api.cc"
 #include "../src/runtime/vulkan/vulkan_module.cc"
 #endif
+
+
+#include <android/log.h>
+
+void dmlc::CustomLogMessage::Log(const std::string& msg) {
+  __android_log_print(ANDROID_LOG_DEBUG, "TVM_RUNTIME", "%s", msg.c_str());
```

Comment:
i guess `__android_log_print(ANDROID_LOG_DEBUG, "TVM_RUNTIME", msg.c_str());` can give us what we want without the `"%s", `?

Reply:
Could simplify like

```
#define LOGD(MSG) __android_log_print(ANDROID_LOG_DEBUG , "TVM_RUNTIME",MSG);

LOGD("Welcome")
```

Reply:
`__android_log_print` assumes the string is a printf-style format string, so just passing in msg wouldn't work if the log message contained any formatting directives. However, there is an `__android_log_write` which does do this. I'll get this changed.

Reply:
@srkreddy1238 I don't think that preserves the functionality of this PR. The intent is to capture all log messages that TVM generates, and pass them on to logcat. Without this, there is no way to observe the log messages generated by the native TVM code in android app (we would otherwise only see messages from java code).